test/mocks.js

```diff
@@ -67,6 +69,72 @@ const kafkaMessagePayloadDto = ({
   }
 })
 
+const kafkaMessagePayloadPostDto = ({
```

Comment:
maybe, we can avoid creating new DTO by "copy/paste", and try to reuse existing DTO:
```
const  kafkaMessagePayloadPostDto = (params = {}) => kafkaMessagePayloadPostDto({
   ...params,
   action: 'post',
   // other fields, specific to POST payload 
})   
```
so it will be much easier to support such code, IMHO